src/flutter-pi.c

```diff
@@ -1680,7 +1686,20 @@ static int init_application(void) {
 	int ok;
 
 	libflutter_engine_handle = NULL;
-	if (flutterpi.flutter.runtime_mode == kRelease) {
+
+    libflutter_engine_handle = dlopen(flutterpi.flutter.libflutter_engine_path, RTLD_LOCAL | RTLD_NOW);
+
+    if (libflutter_engine_handle == NULL)
+    {
+        LOG_FLUTTERPI_ERROR(
+            "[flutter-pi] Warning: Could not load libflutter_engine.so inside the asset bundle : "
+            "%s. Trying to open libflutter_engine.so.${runtimeMode} ...\n",
```

Comment:
`${}` string interpolation came like 50 years after C was invented, so it won't work here I'm afraid ;)

```suggestion
            "%s. Trying to open libflutter_engine.so.%s ...\n",
            dlerror(), flutterpi.flutter.runtime_mode == kDebug ? "debug" : flutterpi.flutter.runtime_mode == kProfile ? "profile" : "debug"
```